### PR TITLE
#163758421 Refactor controllers, models, migrations

### PR DESCRIPTION
**Pull request by erozonachi**
#### What does this PR do?
This PR refactors controllers, models and migrations files

#### Description of Task to be completed?
- ensure response parameter message is changed to error
- ensure to catch all promises
- ensure to replace pg.Client with pg.Pool
- ensure to write a migration to drop tables

#### Screenshots

image

### CODE REVIEW

File: server/models/office.model.js [hunk 1]
```diff
@@ -13,17 +13,16 @@ export default {
...
-      const result = connector.query('INSERT INTO office(type, name, createdOn) values($1, $2, DEFAULT) RETURNING id, type, name, createdOn',
+      const result = connector.query('INSERT INTO office(type, name, createdOn) values($1, $2, DEFAULT) RETURNING office_id, type, name, createdOn',
         [newOffice.type.trim().toLowerCase(), newOffice.name.trim().toLowerCase()]);
 
       result.then((result) => {
         resolve(result);
       }, (error) => {
         reject(error);
-      });
+      }).catch(err => console.error('Error executing query', err.stack));
```
Comment (tutugodfrey) [inline server/models/office.model.js]: Are you supposed to console log the errors or return them?

File: server/controllers/AuthController.js [hunk 2]
```diff
@@ -49,11 +49,11 @@ class AuthController {
     });
     createUser.then((result) => {
       if (result.rowCount <= 0) {
-        return res.status(503).json({ status: 503, error: 'Database connection failed'});
+        return res.status(508).json({ status: 508, error: 'Database connection failed'});
```
Comment (tutugodfrey) [inline server/controllers/AuthController.js]: What is the relationship between rowCount <= 0 and `Database connection failed`. And would you rather think of less technical terms for describing your error messages?

File: server/controllers/AuthController.js [hunk 3]
```diff
@@ -85,7 +85,7 @@ class AuthController {
       } else {
         const user = result.rows[0];
         const userInfo = {
-          id: user.id,
+          id: user.acct_id,
```
Comment (tutugodfrey) [inline server/controllers/AuthController.js]: What is acct_id? Would you rather use a more descriptive name? Also, why are you using snake case? It is not a convention in Javascript.

File: server/models/office.model.js (hunk 1 above)
Comment (erozonachi) [reply server/models/office.model.js]: Console logging is for development purposes, I will replace that with a user-friendly error response before deploying to Heroku

**Pull request merged**